### PR TITLE
Add recipe for limix

### PR DESCRIPTION
**Pull request by Horta**

### CODE REVIEW

File: recipes/limix/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install --compatible --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy
```
Comment (scopatz) [inline recipes/limix/meta.yaml]: If you are going to build against numpy, you need to use `numpy x.x` http://conda-forge.github.io/docs/meta.html#building-against-numpy

**Comment by scopatz** [comment]
LGTM other than the numpy build issue.

File: recipes/limix/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipes/limix/meta.yaml]: Yep. When you have `https://github.com/PMBio/limix/blob/master/setup.py#L142` in your code you need `numpy x.x`, at both `build` and `run` sections, to baked the `numpy` build string in the package name. That ensures the right `numpy` version will be used with the package.

**Comment by ocefpaf** [comment]
The error you are seeing is because some of the dependencies are not available for `numpy 1.12`.
If I had to guess I would say it is either `scikit-learn` or `h5py` (or both).

To fix this we need to: search the channel, find the ones that are missing,  and issue re-rendering there to fix this.

**Comment by Horta** [comment]
https://github.com/conda-forge/h5py-feedstock/issues/17

**Comment by ocefpaf** [comment]
> conda-forge/h5py-feedstock#17

@Horta it is better to fix the missing `numpy 1.12` package with re-renders yourself than trying to work around it. Are you familiar with `conda-smithy`?

You need to:
-  install `conda-smithy` in your root env with: conda install -c conda-forge conda-smithy;
- then issue the `conda smithy rerender` command from the recipe root directory;
- send those changes in a PR and wait for the new packages to show up in the channel.

**Comment by Horta** [comment]
Hi @ocefpaf . Yes, I'm more or less familiar. I just tried to work around it because conda-smithy is not Python 3.6 ready. But I will try again soon...

**Comment by ocefpaf** [comment]
> Hi @ocefpaf . Yes, I'm more or less familiar. I just tried to work around it because conda-smithy is not Python 3.6 ready. But I will try again soon...

You don't need `conda-smithy` for Python 3.6 to do that. Any python version should work for re-rendering.

**Comment by Horta** [comment]
That is what I mean:

```
(default) ~/w/h5py-feedstock> conda smithy rerender -c auto
Traceback (most recent call last):
  File "/Users/horta/anaconda/bin/conda-smithy", line 4, in <module>
    import conda_smithy.cli
ModuleNotFoundError: No module named 'conda_smithy'
(default) ~/w/h5py-feedstock> conda install -c conda-forge conda-smithy -n root
Fetching package metadata .............
Solving package specifications: .


UnsatisfiableError: The following specifications were found to be in conflict:
  - conda-smithy -> python 2.7*
  - python 3.6*
Use "conda info <package>" to see the dependencies for each package.
```

**Comment by ocefpaf** [comment]
Try putting it into an env: `conda create -n _SMITHY -c conda-forge conda-smithy`

(Something is not OK with your root env.)

**Comment by Horta** [comment]
Ready to merge =)
(will re-render np112 later if that is fine...)

**Comment by ocefpaf** [comment]
> will re-render np112 later if that is fine...

Yep. That is fine.

**Comment by Horta** [comment]
I really appreciate that. Sorry for the headaches =)

**Comment by ocefpaf** [comment]
> I really appreciate that. Sorry for the headaches =)

 No problem ;-)

**Comment by Horta** [comment]
I haven't received any e-mail regarding the new feedstock. Is there some delaying issue?

**Pull request merged**